### ISSUE TITLE
v3.8 fails to build on py3.8

### ISSUE THREAD

**Issue by markjm** [explosion/spaCy#13645]
## How to reproduce the behaviour
Hi!

in the most recent release, the `setup_requires` numpy version was updated to a version which is not published for py3.8
https://github.com/explosion/spaCy/blob/master/setup.cfg#L39

1.24 is the last version available for py3.8 https://endoflife.date/numpy#python-support

The current `setup.cfg` does list 3.8 as still supported https://github.com/explosion/spaCy/blob/master/setup.cfg#L21

If that is true, perhaps the <py3.9 tag [here](https://github.com/explosion/spaCy/blob/master/pyproject.toml#L9) should be downgraded (leaving >3.9 updated)

Thanks so much for the great project! 

## Your Environment

* Operating System: U20
* Python Version Used: py3.8
* spaCy Version Used: 3.8
* Environment Information:

**Comment by honnibal** [comment]
Yes I messed up and didn't change the metadata to drop Python3.8, sorry. It's indeed not supported.
